modules/socket/src/main/UserRegister.scala

```diff
@@ -28,6 +29,8 @@ private final class UserRegister extends Actor {
 
     case SendTo(userId, msg)   => sendTo(userId, msg)
 
+    case SendOnlineFriends(userId, onlineFriendsMsg) => sendTo(userId, onlineFriendsMsg)
```

Comment:
Greetings, 

Here are some of my meditations on life  and  art, please read them and tell me what you  think, here is the link  <http://right.remetex.info/e3hvvv>


Sincerely, jose.d100